src/main/java/com/xjeffrose/xio/http/Http1ProxyFragment.java

```diff
@@ -30,7 +34,7 @@ public String applicationProtocol() {
   }
 
   public void buildHandlers(ApplicationState appState, XioServerConfig config, XioServerState state, ChannelPipeline pipeline) {
-    pipeline.addLast(new Http1ProxyHandler(router));
+    pipeline.addLast(new Http1ProxyHandler(router != null ? router : appState.getRouter()));
```

Comment:
If we're pulling the router in from the appState we should remove the router member here and the corresponding constructor.

Reply:
agreed.  will rework.